docs/managing-data-quality/event-recovery-for-bdp-users/manual-event-recovery-for-snowplow-bdp/running/flink/index.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: "Flink"
-date: "2020-08-26"
+date: "2020-04-14"
```

Comment:
Overwriting with @peel’s version. I checked that there were no relevant changes in the BDP version meanwhile.